windows_containers/windows_rn/windows-containers-release-notes-10-15-x.adoc

```diff
@@ -0,0 +1,50 @@
+[id="windows-containers-release-notes-10-15-x"]
+= {productwinc} release notes
+include::_attributes/common-attributes.adoc[]
+:context: windows-containers-release-notes
+
+toc::[]
+
+[id="about-windows-containers"]
+== About {productwinc}
+
+Windows Container Support for Red{nbsp}Hat OpenShift enables running Windows compute nodes in an {product-title} cluster. Running Windows workloads is possible by using the Red{nbsp}Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With Windows nodes available, you can run Windows container workloads in {product-title}.
+
+The release notes for Red{nbsp}Hat OpenShift for Windows Containers tracks the development of the WMCO, which provides all Windows container workload capabilities in {product-title}.
+
+ifndef::openshift-origin[]
+[id="getting-support"]
+== Getting support
+
+// wording taken and modified from https://access.redhat.com/support/policy/updates/openshift#windows
+
+Windows Container Support for Red{nbsp}Hat OpenShift is provided and available as an optional, installable component. Windows Container Support for Red{nbsp}Hat OpenShift is not part of the {product-title} subscription. It requires an additional Red{nbsp}Hat subscription and is supported according to the link:https://access.redhat.com/support/offerings/production/soc/[Scope of coverage] and link:https://access.redhat.com/support/offerings/production/sla[Service level agreements].
+
+You must have this separate subscription to receive support for Windows Container Support for Red{nbsp}Hat OpenShift. Without this additional Red{nbsp}Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red{nbsp}Hat Customer Portal].
+
+For more information, see the Red{nbsp}Hat OpenShift Container Platform Life Cycle Policy document for link:https://access.redhat.com/support/policy/updates/openshift#windows[{productwinc}].
+
+If you do not have this additional Red{nbsp}Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
+endif::openshift-origin[]
+
+[id="about-windows-containers-numbering"]
+== WMCO numbering
+
+Starting with this release, y-stream releases of the WMCO will be in step with {product-title}, with only z-stream releases between {product-title} releases. The WMCO numbering reflects the associated {product-title} version in the y-stream position. For example, the current release of WMCO is associated with {product-title} version 4.15. Thus, the numbering is WMCO 10.15.z.
+
+[id="wmco-10-15-2"]
+== Release notes for Red{nbsp}Hat Windows Machine Config Operator 10.15.2
+
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 10.15.2 were released in link:https://access.redhat.com/errata/RHBA-2024:TBD[RHBA-2024:TBD].
```

Comment:
I am assuming the TBD will be updated when 10.15.2 ships?

Reply:
Yes.